ingenialink/register.py

```diff
@@ -222,6 +231,24 @@ def range(
             return self._range
         return (None, None)
 
+    @cached_property
+    def is_monitoreable(self) -> bool:
+        """True if the register is monitoreable, False otherwise."""
+        return None not in self._monitoring
+
+    @property
+    def monitoring(self) -> Union[tuple[None, None, None], tuple[int, int, RegCyclicType]]:
+        """Containing the address, subnode and cyclic access.
+
+        If the register is not monitoreable, it will contain None.
+
+        Raises:
+            RuntimeError: if monitoring data is invalid.
+        """
+        if self.is_monitoreable and None in self._monitoring:
```

Comment:
If you return either empty values or filled values why do you need to check this?
It's an error that will never appear.

I think it will be far simpler if you have a dataclass dedicated to it.
And return None or dataclass instance here.